utils/build-script-impl

```diff
@@ -2227,8 +2227,8 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
-                if [[ "${SWIFT_THREADING_PACKAGE}" ]] ; then
-                    case "${SWIFT_THREADING_PACKAGE}" in
+                if [[ "${SWIFT_STDLIB_THREADING_PACKAGE}" ]] ; then
```

Comment:
If this is an empty string, I believe bash evaluates this as false and skips this block, so you'll need to correct the list of  cases below.

Reply:
I don't believe this is relevant any more either.